onnxruntime/core/providers/openvino/ov_versions/data_ops.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+#include <unordered_set>
+
+namespace onnxruntime {
+namespace openvino_ep {
+
+typedef enum version_num { 
+  VERSION_2020_4,
+  VERSION_2021_1,
+  VERSION_2021_2
+} version_id_e;
+
+using ConfirmationFunction = std::function<bool(const Node*, const Provider_InitializedTensorSet&)>;
+
+typedef struct supportedop {
+  version_id_e version;
+  std::vector<std::string> device_type;  
+  std::string optype;
+} supportedop_t;
+
+typedef struct unsupportedopmode {
+  version_id_e ver;
+  std::string dev;
+  ConfirmationFunction func; 
+} unsupportedopmode_t;
+
+class Capability{
+
+private:
```

Comment:
Let us add underscore at the end of all private and protected member variables of the class just to adhere with the coding standards
Ex: version_id_e **version_id_;**

Reply:
Resolved